napari_aicssegmentation/core/layer_reader.py

```diff
@@ -31,3 +31,24 @@ def get_channels(self, layer: Layer) -> List[Channel]:
         for index in range(img.shape[index_c]):
             channels.append(Channel(index))
         return channels
+
+    def get_channel(self, index: int):
```

Comment:
This method doesn't look used?

Reply:
I thought we could use this when we set the selected channel in the model

Reply:
It's not needed for that since we are getting the channel from the dropdown list. In general just avoid adding methods that are unused since they likely will not end up being used.